src/App/App.tsx

```diff
@@ -77,6 +95,12 @@ toast.configure({
 });
 initGoogleAnalytics();
 
+const APP_CALLBACK_URL = BACKEND_ACTIVE ? OAUTH_CALLBACK_URL : REACT_LOGIN_URL;
+const { IMPLICIT, AUTHORIZATION_CODE } = AuthorizationGrantType;
+const AuthGrandType = BACKEND_ACTIVE ? AUTHORIZATION_CODE : IMPLICIT;
+const APP_LOGIN_URL = BACKEND_ACTIVE ? LOGIN_URL : REACT_LOGIN_URL;
+const APP_CALLBACK_PATH = BACKEND_ACTIVE ? OAUTH_CALLBACK_PATH : REACT_CALLBACK_PATH;
+
```

Comment:
Can we name these variables in a way that it is easier to differentiate between the REACT_x variables Vs the BACKEND_x variables?  For instance I think `OAUTH_CALLBACK_PATH` is not clear that it relates to the backend while `REACT_CALLBACK_PATH` is very clear that it relates to React/the frontend.